### PR TITLE
Use file attachments for email replies rather than FileComments

### PR DESCRIPTION
**Pull request by adam-thomas**
Follows on from https://github.com/incuna/incuna-groups/pull/43.

@incuna/backend review please?

### CODE REVIEW

File: groups/views/comments.py [hunk 1]
```diff
@@ -126,19 +126,19 @@ def extract_uuid_from_email(email, request):
...
         `x` is a number.
         """
         for attachment in request.FILES.values():
-            models.FileComment.objects.create(
+            models.AttachedFile.objects.create(
                 file=attachment,
                 user=user,
-                discussion=discussion,
+                attached_to=comment,
             )
```
Comment (meshy) [inline groups/views/comments.py]: This could use [`bulk_create`](https://docs.djangoproject.com/en/dev/ref/models/querysets/#bulk-create)

**Comment by meshy** [comment]
Happy to merge as is, happier if nitpick addressed.

**Comment by adam-thomas** [comment]
Changed it :)

**Comment by adam-thomas** [comment]
Travis is saying `ImportError: cannot import name PolymorphicModel` (in `groups.models`) for unknown reasons.  I don't think that's anything to do with my changes...

**Comment by Ian-Foote** [comment]
Are we installing the right version of `django-polymorphic`?

**Comment by adam-thomas** [comment]
@Ian-Foote I fixed that in a different PR once I figured out what was going on.  `polymorphic` 0.8 is broken or has been refactored or something.  There's an issue for it on the `django-polymorphic` repo.

**Pull request merged**